packages/babel-helpers/src/helpers/applyDecs2203.js

```diff
@@ -1,5 +1,10 @@
 /* @minVersion 7.19.0 */
 
+/**
+ * NOTE: This is an old version of the helper, used for 2022-03 decorators.
+ * Updates should be done in applyDecs2203-2.js.
```

Comment:
Should it also be `applyDecs2203R.js` here?

Reply:
```suggestion
 * Updates should be done in applyDecs2203R.js.
```